.github/workflows/tests-unit-php.yml

```
@@ -118,18 +118,14 @@ jobs:
       #
       # | WP  / PHP | PHPUnit |
       # |-----------|---------|
-      # | 5.7 / 7.4 | 7       |
       # | 5.9 / 7.4 | 8       |
       # | 5.9 / 7.4 | 9       |
       # | *   / 8   | 9       |
       #
       # See https://make.wordpress.org/core/handbook/references/phpunit-compatibility-and-wordpress-versions/
       - name: Update PHPUnit
         run: |
-          if [[ $WP_VERSION == "5.7" && $PHP_VERSION == "7.4" ]]; then
-            echo "Installing PHPUnit 7.5.x"
-            composer require --no-interaction --no-scripts phpunit/phpunit:^7.5 --with-dependencies
-          elif [[ $WP_VERSION == "latest" || $WP_VERSION == "trunk" || $PHP_VERSION == "7.4" ]]; then
+          if [[ $WP_VERSION == "latest" || $WP_VERSION == "trunk" || $PHP_VERSION == "7.4" ]]; then
             echo "Installing latest version of PHPUnit"
             composer update --ignore-platform-reqs --no-interaction --no-scripts yoast/phpunit-polyfills --with-dependencies
           fi
```

I don't think the condition is even necessary anymore

```suggestion
          echo "Installing latest version of PHPUnit"
          composer update --ignore-platform-reqs --no-interaction --no-scripts yoast/phpunit-polyfills --with-dependencies
```